path_config.go

```diff
@@ -374,9 +387,40 @@ func (b *jwtAuthBackend) createProvider(config *jwtConfig) (*oidc.Provider, erro
 		supportedSigAlgs = []oidc.Alg{oidc.RS256}
 	}
 
+	var opts []oidc.Option
+	if len(config.UnsupportedCriticalCertExtensions) > 0 {
```

Comment:
If `len(config.UnsupportedCriticalCertExtensions) == 0` and `config.OIDCDiscoveryCAPEM == ""`, then this is all a no-op, and opts is then empty, correct?
Is that the same behavior as before, since you've replaced `oidc.WithProviderCA(config.OIDCDiscoveryCAPEM)` with `opts...`?

Reply:
Yeah, it was weird to me that option was being provided when the config entry was empty.  The underlying field in the cap library is a string, so it's zero value is "", so it's okay to have provided it with the option setting it to "" but not super clean.